phygnn/layers/custom_layers.py

```diff
@@ -615,16 +615,21 @@ class SkipConnection(tf.keras.layers.Layer):
     skip start and skip end.
     """
 
-    def __init__(self, name):
+    def __init__(self, name, method='add'):
         """
         Parameters
         ----------
         name : str
             Unique string identifier of the skip connection. The skip endpoint
             should have the same name.
+        method : str
+            Method to use for combining the skip start data and skip end data.
+            Defaults to 'add'. If 'concat' this is applied along the trailing
+            axis
         """
         super().__init__(name=name)
         self._cache = None
+        self._method = method
```

Comment:
I cant remember exactly how this works but i think this is a breaking change for all pretrained sup3r models that will fail on "layer does not have _method attribute". Can you check to see if there's an easy way to fix this using getattr(self, '_method', 'add')?